nuxeo-management-rest-api/src/test/java/org/nuxeo/rest/management/TestRenditionsObject.java

```diff
@@ -0,0 +1,222 @@
+import org.nuxeo.runtime.test.runner.TransactionalFeature;
+
+import com.fasterxml.jackson.databind.JsonNode;
+
+/**
+ * @since 11.1
+ */
+@RunWith(FeaturesRunner.class)
+@Features(AutomationFeature.class)
+@Deploy("org.nuxeo.ecm.core.management")
+@Deploy("org.nuxeo.ecm.platform.tag")
+@Deploy("org.nuxeo.ecm.platform.picture.api")
+@Deploy("org.nuxeo.ecm.platform.picture.core")
+@Deploy("org.nuxeo.ecm.platform.picture.convert")
+@Deploy("org.nuxeo.ecm.platform.convert")
+@Deploy("org.nuxeo.ecm.platform.types.api")
+@Deploy("org.nuxeo.ecm.platform.types.core")
+@SuppressWarnings("unlikely-arg-type")
+public class TestRenditionsObject extends ManagementBaseTest {
+
+    @Inject
+    protected CoreSession session;
+
+    public static final String PATH = "site/management/renditions";
+
+    protected static final String ENTITY_TYPE = "entity-type";
+
+    protected static final String COMMAND_ID = "commandId";
+
+    @Inject
+    protected TransactionalFeature txFeature;
+
+    @Inject
+    protected ThumbnailService thumbnailService;
+
+    @Test
+    public void testRendtionsPostPicturesRecompute() throws IOException {
+        // test the picture views are computed correctly and trashing them
+        DocumentRef docRef = beforePictureViewsRecompute();
+
+        // generating new picture views
+        runBulkAction("/pictures/recompute");
+
+        // checking the picture views are recomputed in the session
+        afterPicturesRecompute(docRef);
+    }
+
+    @Test
+    @Deploy("org.nuxeo.ecm.platform.thumbnail")
+    public void testRendtionsPostThumbnailsRecomputeBulkStatus() throws IOException {
+        // check the original thumbnail is computed and trashing it
+        DocumentRef docRef = beforeThumbnailsRecompute();
+
+        // generating a new thumbnail
+        runBulkAction("/thumbnails/recompute");
+
+        // checking the thumbnail is recomputed in the session
+        afterThumbnailsRecompute(docRef);
+    }
+
+    protected JsonNode validateBulkStatus(CloseableClientResponse response) throws IOException {
+        assertEquals(HttpServletResponse.SC_OK, response.getStatus());
+        JsonNode node = mapper.readTree(response.getEntityInputStream());
+        JsonNode value = node.get(ENTITY_TYPE);
+        assertEquals(STATUS_ENTITY_TYPE, value.asText());
+        assertNotNull(node.get("state"));
+        assertNotNull(node.get("processed"));
+        assertNotNull(node.get("error"));
+        assertNotNull(node.get("errorCount"));
+        assertNotNull(node.get("total"));
+        assertNotNull(node.get("action"));
+        assertNotNull(node.get("username"));
+        assertNotNull(node.get("submitted"));
+        assertNotNull(node.get("scrollStart"));
+        assertNotNull(node.get("scrollEnd"));
+        assertNotNull(node.get("processingStart"));
+        assertNotNull(node.get("processingEnd"));
+        assertNotNull(node.get("completed"));
+        assertNotNull(node.get("processingMillis"));
```

Comment:
If you just want to check the field's presence it's simpler to do `assertTrue(node.has("FIELD"))`.